Employee/Employee.py

```diff
@@ -66,6 +66,8 @@ def __post_init__(self):
                 else:
                     val = t(self.__getattribute__(field.name))
                 setattr(self, field.name, val)
+        if self.hashed_password == '':
```

Comment:
this is a security vulnerability that was mentioned as fixed in the bug tracker. the database.csv on main already has the passwords set to their employee IDs

Reply:
I never saw that bug tracker, so I'm not sure what you are referring to. The reason I added it in is currently there is no way to add a password via the application to any new employees or imported ones. This means when I tried to add a new user for the professor to use it never had a password set. I understand this could be a security violation, but please understand this is an example or prototype application. If this application were to be released we would need to change a lot of things, but right now there is no other way, which is why I added this line back in. In the future it definitely would be removed, but I don't think it's a good idea to remove it from our prototype right now